Baggage Initial Documentation (#25644)
DataDog/documentation: content/en/tracing/trace_collection/trace_context_propagation/_index.md

- [Datadog][1] (takes higher precedence when extracting headers)
- [W3C Trace Context][2]

> - [Baggage][7]

Reviewer: This link isn't working because the reference for `[7]` is within the `Java` tab. It will need to be moved to the list of reference links at the very bottom, outside of any tabs, on line `651`.

`DD_TRACE_PROPAGATION_STYLE`
: Specifies trace context propagation formats for extraction and injection in a comma-separated list. May be overridden by extract-specific or inject-specific configurations.<br>

> **Default**: `datadog,tracecontext,baggage` <br>

Reviewer 2: I don't think we should change this yet since it's not true for any of the tracers.

Author: Just the default and note below it right?

Author: Or do you mean revert the entire file

Reviewer 2: The change in line 32 above is also "By default, the Datadog SDK extracts and injects distributed tracing headers using the following formats," which is not available yet. 😬

Reviewer 2: I would say, don't add anything regarding configuration just yet. `:2cents:`

| [W3C Trace Context][2] | `tracecontext`                |
| [B3 Single][3]         | _Language Dependent Value_    |
| [B3 Multi][4]          | `b3multi`                     |

> | [Baggage][7]           | `baggage`                     |

Reviewer: This link is not working for the same reason mentioned above

[4]: https://github.com/openzipkin/b3-propagation#multiple-headers
[5]: https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-tracingheader
[6]: #none-format

> [7]: https://www.w3.org/TR/baggage/

Reviewer: ```suggestion
```

Reviewer: This link will need to move outside of the tab structure to line `651` in order to work

When the Datadog SDK is configured with the None format for extraction or injection (possibly both), the Datadog SDK does _not_ interact with request headers, meaning that the corresponding context propagation operation does nothing.

### Baggage format

> Baggage propagation using an OpenTelemetry/W3C-compatible header will be enabled by default. The SDKs allow disabling the baggage propagator by explicitly configuring the settings above and not including the baggage header. 

Reviewer: ```suggestion
Baggage propagation through OpenTelemetry's W3C-compatible headers is enabled by default. The SDKs allow disabling the baggage propagator by explicitly configuring the settings above and not including the baggage header. 
```
